spec/codebreaker/guess_checker_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+RSpec.describe Codebreaker::GuessChecker do
+  test_data = [
```

Comment:
let

Reply:
after this array i do let(:test_data)  { test_data} - need help cuz when i try to do let(:test_data) without locale variable
rubocop  quarrel with me. I can't  create let with multiple line. Also i  tried to create an additional folder where I would place this local variable and then require_relative to this file but it doesn't work for me) Need help

Reply:
Exclude for this file